plugin/src/main/java/com/playmonumenta/scriptedquests/point/Point.java

```diff
@@ -7,26 +7,35 @@ public class Point {
 	public double mX;
 	public double mY;
 	public double mZ;
+	public float mYaw;
+	public float mPitch;
 
 	public Point(double x, double y, double z) {
+		this(x, y, z, 0, 0);
+	}
+
+	public Point(double x, double y, double z, float yaw, float pitch) {
 		mX = x;
 		mY = y;
 		mZ = z;
+		mYaw = yaw;
+		mPitch = pitch;
 	}
 
 	public Point(Location loc) {
 		mX = loc.getX();
 		mY = loc.getY();
 		mZ = loc.getZ();
+		mYaw = loc.getYaw();
+		mPitch = loc.getPitch();
 	}
 
 	@Override
 	public String toString() {
-		return "(" + Double.toString(mX) + ", " +
-			Double.toString(mY) + ", " + Double.toString(mZ) + ")";
+		return "(" + mX + ", " + mY + ", " + mZ + ")";
```

Comment:
shouldn't this include yaw and pitch?

Reply:
Technically, I guess? The places where Points are toString'ed doesn't care about yaw and pitch though (I maybe should have made a new class instead of extending this one, idk)